awx/ui/client/src/notifications/notificationTemplates.form.js

```diff
@@ -128,10 +128,25 @@ export default ['i18n', function(i18n) {
                     reqExpression: "port_required",
                     init: "false"
                 },
-                ngShow: "notification_type.value == 'email' || notification_type.value == 'irc'",
+                ngShow: "notification_type.value === 'email' || notification_type.value === 'irc'",
                 subForm: 'typeSubForm',
                 ngDisabled: '!(notification_template.summary_fields.user_capabilities.edit || canAdd)'
             },
+            timeout: {
+                label: i18n._('Timeout'),
+                type: 'number',
+                integer: true,
+                spinner: true,
+                min: 0,
+                default: 0,
```

Comment:
should the default timeout be 0? Seems like very little time

Reply:
It seems that a timeout value of 0 = "no timeout".  Still testing this, however.